design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,212 @@
+    samples: [DownloadableProductSamples] @doc(description:  "DownloadableProductSamples defines characteristics of a downloadable product")
+}
+
+type BundleRequisitionListItem implements NegotiableQuoteItemInterface
+@doc(description: "Negotiable Quote Item Implementation that for Bundle Products") {
+    id: ID! @doc(description: "Unique Identifier of Negotiable Quote Item")
+    product: ProductInterface!
+    qty: Float! @doc(description: "Quantity added")
+    customizable_options: [SelectedCustomizableOption] @doc(description: "custom Option selected")
+    bundle_options: [SelectedBundleOption]! @doc(description: "selected bundle options")
+}
+
+type ConfigurableRequisitionListItem implements NegotiableQuoteItemInterface
+@doc(description: "Negotiable Quote Item Implementation that for Configurable Products") {
+    id: ID! @doc(description:  "Unique Identifier of Negotiable Quote Item")
+    product: ProductInterface!
+    qty: Float! @doc(description: "Quantity added")
+    customizable_options: [SelectedCustomizableOption] @doc(description: "custom Option selected")
+    configurable_options: [SelectedConfigurableOption] @doc(description: "Configurable options selected")
+}
+
+type AttachmentContent @doc(description: "Negotiable quote attachment file") {
+    base64_encoded_data: String!
+    mime_type: String!
+    name: String!
+}
+
+type NegotiableQuoteComment {
+    id: ID!
+    parent_id: String!
+    author: String!
+    text: String
+    created_at: String
+    attachments: [String]
+}
+
+type NegotiableQuoteHistoryLog {
+    id: ID!
+    author: String!
+    action: String!
+    comment: String
+    status: String
+    created_at: String
+}
+
+type NegotiableQuotePaymentInfo {
+    payment_methods: [PaymentMethodsOutput]! @doc(description: "All list of payment options and totals")
+    prices: CartPrices! @doc(description: "List of totals")
+}
+
+type PaymentMethodsOutput {
+    code: String @doc(description: "Payment method code")
+    title: String @doc(description: "Payment method title")
+}
+
+type CreateNegotiableQuoteOutput {
+    negotiable_quote:NegotiableQuote!
+}
+
+type AddNegotiableQuoteCommentsOutput {
+    negotiable_quote:NegotiableQuote!
+}
+
+type UpdateNegotiableQuoteItemsQuantityOutput {
+    negotiable_quote:NegotiableQuote!
+}
+
+type RemoveNegotiableQuoteItemsOutput{
+    negotiable_quote:NegotiableQuote!
+}
+
+input FileInput @doc(description: "The list of file attachment codes") {
+    base64_encoded_data: String!
+    mime_type: String!
+    name: String!
+}
+
+input NegotiableQuoteShippingAddressInput {
+    address: CartAddressInput
+}
+
+type ShippingMethodsOutput {
+    carrier_code: String @doc(description: "Shipping carrier code")
+    method_code: String @doc(description: "Shipping method code")
+    carrier_title: String @doc(description: "Shiping carrier title")
+    method_title: String @doc(description: "Shipping method title")
+    amount: String @doc(description: "Shipping amount")
```

Comment:
Don't think a monetary value as a `String` is going to work (quite a few fields on this type treating money as `String`)